test/test_jit.py

```diff
@@ -1362,7 +1366,7 @@ def f(x):
             out.copy_(x)
```

Comment:
(line 1350) You deleted test_alexnet.expect but it doesn't look like you updated the test. Is the test dead?

Reply:
Ditto for test_batchnorm and test_conv, what am I missing?

Reply:
The tests were updated in a different PR & i forgot to remove the expect - will rebase PR in a sec so it makes sense